templates/ecs-asg-lch.template.yaml

```
@@ -492,4 +492,7 @@ Outputs:
     Value: !Ref TaskDefinition
   ECSALB:
     Description: ALB DNS URL
-    Value: !GetAtt ALBResource.DNSName
\ No newline at end of file
+    Value: !GetAtt ALBResource.DNSName
+  ECSASGStackName:
+    Description: Stack Name of the current stack
```

Minor suggested correction to use sentence case.
```suggestion
    Description: Stack name of the current stack
```